### PR TITLE
(haali-media-splitter) Moved to manual+Modernized

### PR DESCRIPTION
**Pull request by AdmiringWorm**
requires `[PUSH haali-media-splitter]` on merge.

### CODE REVIEW

File: manual/haali-media-splitter/haali-media-splitter.nuspec [hunk 1]
```diff
@@ -1,23 +1,26 @@
...
 <!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
-<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
-    <id>{{PackageName}}</id>
-    <version>{{PackageVersion}}</version>
-    <title>Haali Media Splitter</title>
+    <id>haali-media-splitter</id>
+    <version>1.13.138.1401</version>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/haali-media-splitter</packageSourceUrl>
```
Comment (pascalberger) [inline manual/haali-media-splitter/haali-media-splitter.nuspec]: Should be `/manual/`

Comment (AdmiringWorm) [inline manual/haali-media-splitter/haali-media-splitter.nuspec]: ops, thanks. I'll get that fixed.

**Comment by AdmiringWorm** [comment]
@pascalberger I've updated the package source url.

**Comment by pascalberger** [comment]
@AdmiringWorm Thanks LGTM

**Pull request merged**